_posts/2017-08-23-ce-gsoc-summary.md

```diff
@@ -0,0 +1,116 @@
+---
+layout:     post
+title:      Custom Elements in Servo
+date:       2017-08-23 20:00:00
```

Comment:
I think this maybe should be changed to the 24th. Currently it comes after another blog post that says it was published on the 24th making the ordering look messed up.